ingestion/functions/parsing/japan/japan_test.py

```diff
@@ -24,6 +29,15 @@
                             "start": "03/13/2020Z",
                             "end": "03/13/2020Z"
                         }
+            },
+            {
+                "name": "outcome",
+                "dateRange":
```

Comment:
as per my other comment, you should'nt put a dateRange with null start and end fields if the outcome is unknown